### PR TITLE
Added some commishes

### PR DESCRIPTION
**Pull request by BrownDairy**

### CODE REVIEW

File: cardlibrary/10.12.16.lua [hunk 1]
```diff
@@ -1432,7 +1432,96 @@ local tentwelvesixteen = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
...
+            	["Cost"] = {["Neutral"] = 2, ["Red"] = 1,},
+            	["Effect"] = {
+                  	Name = "Hellfire",
+                   	Description = "Can only be summoned if your opponent has 2 or more fighters. Whenever you summon a fighter, TheJazzCat22 gains 150 power and 150 health.",
+                    	["Type"] = "OnAllySummon",
+                    	["Power"] = {{"Strengthen",150},{"Heal",150}},
+                    	Target = "Self",
+                },
+           	["Bio"] = "Ya' like jazz?",
+        }
```
Comment (WoodsmanTT) [inline cardlibrary/10.12.16.lua]: Don't you think this needs a comma?

File: cardlibrary/10.12.16.lua [hunk 2]
```diff
@@ -1432,7 +1432,96 @@ local tentwelvesixteen = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO
             		Target = "ColorNeutral",
        	 	},
         	["Bio"] = "He certainly isn't.",
+    	},
+	
+	["emilvita"] = {
+           	["Id"] = 679503783,
```
Comment (WoodsmanTT) [inline cardlibrary/10.12.16.lua]: Provide the correct art pls

Comment (BrownDairy) [inline cardlibrary/10.12.16.lua]: where did i miss the comma

**Comment by BeligerentBanana** [comment]
Emil must be given a new effect

**Pull request merged**